docs/en/api/promql-service.md

```diff
@@ -0,0 +1,632 @@
+For example: select metric `service_cpm` which the service is `$service` and the layer is `$layer` within the last 5 minutes.
+```shell
+service_cpm{service='$service', layer='$layer'}[5m]
+```
+
+#### [Time Durations](https://prometheus.io/docs/prometheus/latest/querying/basics/#time-durations)
+| Unit | Definition   | Support |
+|------|--------------|---------|
+| ms   | milliseconds | yes     |
+| s    | seconds      | yes     |
+| m    | minutes      | yes     |
+| h    | hours        | yes     |
+| d    | days         | yes     |
+| w    | weeks        | yes     |
+| y    | years        | **no**  |
+
+### Binary operators
+#### [Arithmetic binary operators](https://prometheus.io/docs/prometheus/latest/querying/operators/#arithmetic-binary-operators)
+| Operator | Definition           | Support |
+|----------|----------------------|---------|
+| +        | addition             | yes     |
+| -        | subtraction          | yes     |
+| *        | multiplication       | yes     |
+| /        | division             | yes     |
+| %        | modulo               | yes     |
+| ^        | power/exponentiation | **no**  |
+
+##### Between two scalars
+For example:
+```shell
+1 + 2
+```
+
+##### Between an instant vector and a scalar
+For example:
+```shell
+service_cpm{service='$service', layer='$layer'} / 100
+```
+
+##### Between two instant vectors
+For example:
+```shell
+service_cpm{service='$service', layer='$layer'} + service_cpm{service='$service', layer='$layer'}
+```
+
+**Note: The operations between vectors require the same metric and labels, and don't support [Vector matching](https://prometheus.io/docs/prometheus/latest/querying/operators/#vector-matching).**
+
+#### [Comparison binary operators](https://prometheus.io/docs/prometheus/latest/querying/operators/#comparison-binary-operators)
+| Operator | Definition       | Support |
+|----------|------------------|---------|
+| ==       | equal            | yes     |
+| !=       | not-equal        | yes     |
+| \>       | greater-than     | yes     |
+| <        | less-than        | yes     |
+| \>=      | greater-or-equal | yes     |
+| <=       | less-or-equal)   | yes     |
+##### Between two scalars
+For example:
+```shell
+1 > bool 2
+```
+
+##### Between an instant vector and a scalar
+For example:
+```shell
+service_cpm{service='$service', layer='$layer'} > 1
+```
+
+##### Between two instant vectors
+For example:
+```shell
+service_cpm{service='service_A', layer='$layer'} > service_cpm{service='service_B', layer='$layer'}
+```
+
+### HTTP API
+
+#### Expression queries
+
+##### [Instant queries](https://prometheus.io/docs/prometheus/latest/querying/api/#instant-queries)
+```shell
+GET|POST /api/v1/query
+```
+
+| Parameter | Definition                                                                                                                          | Support | Optional   |
```

Comment:
Could you explain what `Optional` mean? The context seems not very clear.

Reply:
In the Prometheus official doc, use `Optional` so I think follow this is better.

Reply:
OK, got it.